lib/tasks/next_available_error.rake

```diff
@@ -0,0 +1,12 @@
+namespace :err do
```

Comment:
What do you think about changing the command you run to `error_code:next` instead?

Reply:
Makes sense